### ISSUE TITLE
[Bug Report][3.1.1] Violation : This INPUT has a mismatch between the value of its readonly and aria-readonly attributes

### ISSUE THREAD

**Issue by KhaderShaikPenn** [vuetifyjs/vuetify#16632]
### Environment
**Vuetify Version:** 2.6.2
**Vue Version:** 2.6.10
**Browsers:** Chrome 107.0.0.0
**OS:** Windows 10

### Steps to reproduce
Please ignore the Reproduction link given.  Details as below:
Violation : This INPUT has a mismatch between the value of its readonly and aria-readonly attributes Best Practice : Ensure ARIA roles, states, and properties are valid
<input autocomplete="off" aria-readonly="false" type="text" readonly="readonly" id="input-543" aria-label="Select a County" required="required">

Above is the DOM rendered output. As you see aria-readonly is marked as false while readonly attribute is set as readonly which mismatch . 
This is reported by Level Access tool - Continuum Explorer Pro which is a Chrome extension
The above issue is on all v-select and v-autocomplete components

### Expected Behavior
No Accessibility concern
!

### Actual Behavior
No Accessibility concern

### Reproduction Link
[https://codesandbox.io/s/vue?file=/src/main.js ](https://codesandbox.io/s/vue?file=/src/main.js )

### Other comments
This is with state organization application and can't provide details here. The application in which this is reported is https://apply.findmyridesyst.penndot.pa.gov/  ( providing SYST environment). Need registration to access this application.  Please email me to get more details  as c
